music_album.rb

```diff
@@ -0,0 +1,17 @@
+require_relative './item'
+require 'date'
+
+class MusicAlbum < Item
+  attr_reader :id, :on_spotify
+
+  def initialize(publish_date, on_spotify)
+    super()
+    @id = Random.rand(1..200)
+    @on_spotify = on_spotify
+    @publish_date = Date.parse(publish_date)
+  end
+
+  def can_be_archived?
+    true if super || on_spotify == true
+  end
```

Comment:
- [ ] this method should return true if the parent's method  returns true AND if on_spotify equals true

Reply:
Done 
thank you